q2_types/feature_data/_transformer.py

```diff
@@ -644,3 +644,17 @@ def _224(data: pd.DataFrame) -> DifferentialFormat:
     ff = DifferentialFormat()
     qiime2.Metadata(data).save(str(ff))
     return ff
+
+
+# blast types
+@plugin.register_transformer
+def _225(ff: BLAST6Format) -> pd.DataFrame:
+    return skbio.read(str(ff), format='blast+6', into=pd.DataFrame,
+                      default_columns=True)
+
+
+@plugin.register_transformer
+def _226(data: pd.DataFrame) -> BLAST6Format:
+    ff = BLAST6Format()
+    data.to_csv(str(ff), sep='\t', header=False, index=False)
+    return ff
```

Comment:
quick q : what do you think about adding metadata conversions? It'll basically be

```

@plugin.register_transformer
def _227(ff: BLAST6Format) -> qiime2.Metadata:
    return qiime2.Metadata.load(str(ff))
```

Reply:
@mortonjt Thanks for reviewing this PR! Converting BLAST result into metadata is an interesting idea, but there is a difficulty: In a BLAST hit table, query sequences (row heads) are not unique. The same query can happen multiple times, with different subjects. Therefore, this structure isn't compatible with `qiime2.Metadata`. Even if queries are unique, I still don't see how this structure may be useful as a metadata table. That's because BLAST parameters are calculated for each query-subject pair, and they are not comparable across queries (they are comparable across subjects though).